topic_message_query.go

```diff
@@ -27,11 +27,13 @@ type TopicMessageQuery struct {
 }
 
 func NewTopicMessageQuery() *TopicMessageQuery {
+	timeNow := time.Now()
 	return &TopicMessageQuery{
 		maxAttempts:       maxAttempts,
 		errorHandler:      _DefaultErrorHandler,
 		retryHandler:      _DefaultRetryHandler,
 		completionHandler: _DefaultCompletionHandler,
+		startTime:         &timeNow,
```

Comment:
This shouldn't be necessary. The server will default startTime to now if not sent. The problem must lie elsewhere. Do you have tests for this?

Reply:
Yed we had an e2e test, but it was actually erroring in the end with timeout. So far I am not sure at all what causes it, start time is nil whole time if not set. But if i just set it to empty it works.